### ISSUE TITLE
[Feature Request] Offer `-random` option on the CLI

### ISSUE THREAD

**Issue by jordan-ivpn** [ivpn/desktop-app#144]
# Feature request

## Description

`ivpn connect -any US` always connects to **us-tx** because that is the first server in the servers.json file.

`ivpn connect -p ovpn -any US` always connects to **us-ca** because that is the first server in the OpenVPN section of the servers.json file.

Adding a `-random` CLI option offers a way to randomize the `-any` option for countries with multiple locations, like US, UK, and Canada.

## Describe alternatives you've considered

Scripting a custom wrapper for the IVPN App to select a random server is currently the only option to achieve this behaviour.

**Comment by stenya** [comment]
I see having both options ('-any' and '-random') is redundant.
I think it will be better just to change behavior or `-any` option: it will randomize server selection (currently it takes a first found server)

The option description will be changed from 
`When LOCATION points to more than one server, use first found server to connect` 
to something like 
`When LOCATION points to more than one server, use a random server of the found results to connect`

@jordan-ivpn @gorkapernas What do you think?

**Comment by gorkapernas** [comment]
@stenya agreed, changing the behavior (and description of the command `-any`) would be sufficient instead of adding a new command

**Comment by stenya** [comment]
@jordan-ivpn @gorkapernas 
Ok, I am going to make changes with '-any' option.

We should keep in mind that in this case, a user must use a more specific filtering value.
For example, "US" as a filter fits also to  Br**us**sels and A**us**tria

So, `ivpn connect -any US` will not always connect to United States server.

```
ivpn servers -p wg US
  PROTOCOL|           LOCATION|                    CITY|        COUNTRY| IPv? tunnel|
 WireGuard|  us-tx.wg.ivpn.net|         Dallas, TX (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-il.wg.ivpn.net|        Chicago, IL (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-ga.wg.ivpn.net|        Atlanta, GA (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-ca.wg.ivpn.net|    Los Angeles, CA (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-ny.wg.ivpn.net|       New York, NY (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-ut.wg.ivpn.net| Salt Lake City, UT (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-fl.wg.ivpn.net|          Miami, FL (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-nj.wg.ivpn.net|     New Jersey, NJ (US)|  United States|   IPv4/IPv6|
 WireGuard|     at.wg.ivpn.net|             Vienna (AT)|        Austria|   IPv4/IPv6|
 WireGuard|     be.wg.ivpn.net|           Brussels (BE)|        Belgium|   IPv4/IPv6|
 WireGuard|  us-az.wg.ivpn.net|        Phoenix, AZ (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-nv.wg.ivpn.net|      Las Vegas, NV (US)|  United States|   IPv4/IPv6|
 WireGuard| au-nsw.wg.ivpn.net|             Sydney (AU)|      Australia|   IPv4/IPv6|
 WireGuard|  us-va.wg.ivpn.net|        Ashburn, VA (US)|  United States|   IPv4/IPv6|
 WireGuard|  us-wa.wg.ivpn.net|        Seattle, WA (US)|  United States|   IPv4/IPv6|
```

**Comment by stenya** [comment]
@gorkapernas v3.9.26

**Comment by stenya** [comment]
As @gorkapernas suggested,
it would be nice to have 2 improvements:
1) ability to use `-any` standalone. So, it will be similar to "Random server" in GUI

**Comment by stenya** [comment]
Updated also description for '-any' option 
from `When LOCATION points to more than one server, use a random server from the found results to connect`
to `Use a random server from the found results to connect`

**Comment by stenya** [comment]
v3.9.28

**Comment by gorkapernas** [comment]
Verified in all platforms, latest beta 3.9.29, the option `-any` has been updated according to the last comments, as well as the description of the command.
Currently it also behaves similar to the option `random` from the GUI. 
See previous comments for further details.

**Comment by stenya** [comment]
Released in v3.9.32
